src/genie/libs/parser/iosxe/tests/test_show_cts.py

```diff
@@ -0,0 +1,167 @@
+import unittest
+from unittest.mock import Mock
+
+from genie.metaparser.util.exceptions import SchemaEmptyParserError
+from genie.libs.parser.iosxe.show_cts import Show_Cts_Sxp_Connections_Brief
+
+
+# ==============================================
+# Unit test for 'show_cts_sxp_connections_brief'
+# ==============================================
+class test_show_cts_sxp_connections_brief(unittest.TestCase):
+    """Unit test for 'show_cts_sxp_connections_brief'"""
+
+    maxDiff = None
+    empty_output = {'execute.return_value': ''}
+    golden_parsed_output1 = {
+    "sxp_connections": {
+        "status": {
+            "sxp_status": "Enabled",
+            "highest_version": 4,
+            "default_pw": "Set",
+            "key_chain": "Not Set",
+            "key_chain_name": "Not Applicable",
+            "source_ip": "192.168.2.24",
+            "conn_retry": 120,
+            "reconcile_secs": 120,
+            "retry_timer": "not running",
+            "seq_export": "Not Set",
```

Comment:
I understand, tried to put the short name and it's meaningful at the same time. But simply below one is OK.
```
'peer_sequence_traverse_limit_for_export': 'Not Set'
```
The issue with short one, if something new similar lines are added in future, the short one might not work well. (hard to distinguish which one between similar ones)

Reply:
These keys have been renamed.

Reply:
confirmed